### PR TITLE
Parameters guide and k8s readme modifications

### PR DESCRIPTION
**Pull request by Mionsz**
Parameters guide and k8s readme modifications.
- added docs/ffmpeg-params-guide.md for describsion of all possible parameters in FFmpeg plugin.
- removed not actual Docker build  informations
- added reference to the Tiber project.

### CODE REVIEW

File: docs/ffmpeg-params-guide.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+- **Default**: `0`
+- **Flags**: Encoding parameter
+- **Range**: `-1` to `INT_MAX`
+
+### Example Usage
+
+To use the Mesh's Muxer plugin with FFmpeg, pass the `-f mcm` flag and parameters. Here is an example command that sets some of the parameters:
+
+```bash
+ffmpeg -i input.mp4 -c:v rawvideo -f mcm -ip_addr 192.168.1.100 -port 8000 -payload_type st20 -pixel_format yuv422p10le -protocol_type auto -video_size hd720 -frame_rate 30 -socket_name my_socket -interface_id 2 output.mcm
```
Comment (ko80) [inline docs/ffmpeg-params-guide.md:110]: @Mionsz What output do you expect to be written to `output.mcm`? I don't get it

File: docs/ffmpeg-params-guide.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+- **Type**: String
+- **Default**: `"9001"`
+- **Flags**: Encoding parameter
+
+#### `payload_type`
+- **Description**: Set the payload type for the media data.
+- **Type**: String
+- **Default**: `"st20"`
+- **Flags**: Encoding parameter
+- **Possible Values**: `"st20"`, `"st22"`, ~~`"st30"`~~, ~~`"st40"`~~, ~~`"rtsp"`~~
```
Comment (ko80) [inline docs/ffmpeg-params-guide.md:64]: @Mionsz Only st20 and st22 are supported at this moment.

File: docs/ffmpeg-params-guide.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
...
+
+#### `protocol_type`
+- **Description**: Set the protocol type for the media data transmission.
+- **Type**: String
+- **Default**: `"auto"`
+- **Flags**: Encoding parameter
+- **Possible Values**: `"memif"`, `"udp"`, `"tcp"`, ~~`"http"`~~, ~~`"grpc"`~~, `"auto"`
```
Comment (ko80) [inline docs/ffmpeg-params-guide.md:71]: @Mionsz Only memif, tcp, and auto are supported

File: docs/ffmpeg-params-guide.md (hunk 2 above)
Comment (Mionsz) [reply docs/ffmpeg-params-guide.md:64]: There is a strikethrough 
image

File: docs/ffmpeg-params-guide.md (hunk 1 above)
Comment (Mionsz) [reply docs/ffmpeg-params-guide.md]: fixed.

File: docs/ffmpeg-params-guide.md (hunk 3 above)
Comment (Mionsz) [reply docs/ffmpeg-params-guide.md]: true. There is a strikethrough

**Review by Sakoram — APPROVED** [review]
LGTM

**Pull request merged**